syntax = "proto3";

package search;

service Search{
 rpc GetServerResponse(Message) returns (SearchResults) {}
}

message Message{
 string message = 1;
}

message SearchResults{
  repeated Product product = 1;
}

message Product{
 int32 id = 1;
 string brand_name = 2;
 string items_description = 3;
 float prices = 4;
 string category = 5;
}